trieste/acquisition/rule.py

```diff
@@ -330,7 +330,7 @@ def acquire(
                             )
                         else:  # vectorized acquisition function
                             logging.histogram(
-                                f"EGO.acquisition_function/maximum_found[{i+1}]", values
+                                f"EGO.acquisition_function/maximums_found[{i+1}]", values
```

Comment:
a bit confused here - I would have thought that `[{i+1}]` bit is part of the name as well, and here the naming would be different than what we have above?

Reply:
I'm confused by your confusion :-) Both the scalar and historgram were previously called `maximum_found[{i+1}]`.